resources/translations/en.edn

```diff
@@ -110,7 +118,8 @@
                    :end "End"
                    :errors {:dependencies-archived "Archived dependencies:"
                             :in-use-by "It is in use by:"
-                            :dependencies-not-found "Missing dependencies:"}
+                            :dependencies-not-found "Missing dependencies:"
+                            :disallowed-subcategory "Disallowed subcategories:"}
```

Comment:
We could perhaps be more explicit here by saying "can't be a child" or we could make the checking more generic by checking also that there will be no loop in the category tree. I am wondering if this error is clear when it happens.

Reply:
specified the error so that it should indicate category cannot be set as it's own subcategory